analyses/tp53_nf1_score/run_classifier.sh

```diff
@@ -72,8 +72,8 @@ Rscript --vanilla ${analysis_dir}/00-tp53-nf1-alterations.R \
   --gencode ${cds_file}
 
 # If running for the purpose of figure generation (RUN_FOR_FIGURES will be 1),
-# use the data in the analysis directory that has been freshly collapsed
-if [[ "$RUN_FOR_FIGURES" -eq "0" ]]
+# use the data in the data release directory
+if [[ "$RUN_FOR_FIGURES" -eq "1" ]]
```

Comment:
If you use the data release for figures, you don't need this at all. It was intended to let you use _results_ for figures, as that was an existing paradigm that we've since moved away from.